### ISSUE TITLE
[porter/wordpress] fails with InvalidTemplate error

### ISSUE THREAD

**Issue by squillace** [Azure/azure-cnab-quickstarts#33]
## Expected Behavior
deploys

## Actual Behavior
fails with:
```
{"telemetryId":"e781a3a9-5638-449d-9359-d9a4bda10cb3","bladeInstanceId":"Blade_9cf883664a894b2299414210e5e35460_0_0","galleryItemId":"Microsoft.Template","createBlade":"DeployToAzure","code":"InvalidTemplate","message":"Deployment template language expression evaluation failed: 'Unable to parse language expression '': expected token 'Identifier' and actual 'EndOfData'.'. Please see https://aka.ms/arm-template-expressions for usage details.","additionalInfo":[{"type":"TemplateViolation","info":{"lineNumber":0,"linePosition":0,"path":""}}]}
```

## Steps to Reproduce the Problem

  1. precisely the same as #32

I'll retest both on mac and windows.

**Comment by squillace** [comment]
![image](https://user-images.githubusercontent.com/1422718/70832163-64c95b00-1dc2-11ea-95f6-5ec040ff1979.png)

**Comment by MikeLarah** [comment]
Same issue/fix as #32
